### DISCUSSION TITLE
Footer的高度调整

### DISCUSSION THREAD

**Discussion by leimao** [ppoffice/hexo-theme-icarus#1138] [Q&A] [answered]
请问，这个footer的高度该如何调整，谢谢。
![Screenshot from 2022-10-18 19-13-22](https://user-images.githubusercontent.com/17606112/196581176-5c01327f-c05e-4ef0-a336-3424efe37bca.png)

**Comment by ppoffice** [comment, marked as answer]
[`base.styl`](https://github.com/ppoffice/hexo-theme-icarus/blob/5.1.0/include/style/base.styl)中添加[`$footer-padding`](https://github.com/groenroos/bulma-stylus/blob/v0.8.0/stylus/layout/footer.styl)，例如`$footer-padding ?= 3rem 1.5rem 6rem`。

**Reply by leimao** [reply]
谢谢。
